core/domain/services/contexts/RequestTypeContextDetector.php

```diff
@@ -124,6 +115,36 @@ public function detectRequestTypeContext()
     }
 
 
+    /**
+     * @return RequestTypeContext
+     */
+    private function isAjaxRequest()
+    {
+        if (
+            $this->request->getRequestParam('ee_front_ajax', 0, 'int')
+            || $this->request->getRequestParam('data[ee_front_ajax]', 0, 'int')
+        ) {
+            if (! defined('EE_FRONT_AJAX')) {
+                define( 'EE_FRONT_AJAX', true);
```

Comment:
```suggestion
                define('EE_FRONT_AJAX', true);
```

🙄

Reply:
thnx, will fix!

Reply:
huh... already fixed...

github must have been showing you guys an old file

Reply:
Seems like it 🤔

Reply:
the github carrier pigeons with the code updates must have run into a storm or something?